packages/vanilla/src/sass/components/tabs/_index.scss

```diff
@@ -0,0 +1,58 @@
+.cbp-tabset {
+  display: flex;
+  align-items: flex-end;
+  //gap: var(--cbp-space-2x);
+  border-bottom: var(--cbp-border-size-md) solid var(--cbp-color-gray-cool-30);
+  margin-bottom: var(--cbp-space-4x);
+
+  .cbp-tab {
+    all: unset;
+    min-height: var(--cbp-space-14x);
```

Comment:
The height I'm getting when inspecting the tab is 59px, should be 56px. I think this has to do with the `all: unset` property because I'm seeing that the tabs have the `box-sizing: content-box` property being set in the Computed tab of the inspector.

Reply:
Ah, good catch. I missed that. Also a reason I rarely use `all: unset`. I'll revisit and see what's more efficient.